src/selection-sharer.js

```diff
@@ -232,6 +232,7 @@
     this.shareFacebook = function(e) {
       e.preventDefault();
       var text = self.htmlSelection.replace(/<p[^>]*>/ig,'\n').replace(/<\/p>|  /ig,'').trim();
+      text = text.replace(/<\/?[^>]+(>|$)/g, ""); // Remove HTML tags, like <h3>foo</h3>bar => foobar
```

Comment:
Yep, make sense. Using `textSelection` makes more sense than `htmlSelection`, and I guess it'll solve the problem anyway.